pkgs/applications/audio/emacspeak/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchgit, rsync, gnused, emacs, tcl, espeak-classic }:
+
+stdenv.mkDerivation rec {
+  pname = "emacspeak";
+  version = "50.0";
+
+  src = fetchgit {
```

Comment:
please use fetchFromGitHub, it's less expensive than pulling down an entire git history